projects/driver_controls/test/test_mech_brake.c

```diff
@@ -0,0 +1,70 @@
+#include <stdint.h>
+#include <stdio.h>
+
+#include <stdbool.h>
+#include "adc.h"
```

Comment:
You should only be including the bare minimum required for a test. (In fact, you should only be including the bare minimum required in any module.)